dev/TabView/TabViewItem.cpp

```diff
@@ -232,17 +315,32 @@ void TabViewItem::OnPointerEntered(winrt::PointerRoutedEventArgs const& args)
 {
     __super::OnPointerEntered(args);
 
+    m_isPointerOver = true;
+
     if (m_hasPointerCapture)
     {
         m_isMiddlePointerButtonPressed = true;
     }
+
+    // If we show close button onhover, we now must show the close button
```

Comment:
Remove the if and just call UpdateCloseButton(). All the logic should be there to handle it.